src/Translator/doc/index.rst

```diff
@@ -98,6 +98,13 @@ By default, the default locale is ``en`` (English) that you can configure throug
 #. Or with ``<html data-symfony-ux-translator-locale="your-locale">`` attribute
 #. Or with ``<html lang="your-locale">`` attribute
 
+Detecting missing translations on the front-end
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+By default, the translator will return the translation key as is when the translation is missing.
+
+You can configure this behavior to throw an error instead by calling ``throwWhenNotFound(true)`` from ``@symfony/ux-translator`` package.
```

Comment:
I think it would be more friendly to display the function call like this:
```suggestion
You can configure this behavior to throw an error instead by calling ``throwWhenNotFound(true)``:

.. code-block:: diff

    // assets/translator.js

    /*
     * This file is part of the Symfony UX Translator package.
     *
     * If folder "../var/translations" does not exist, or some translations are missing,
     * you must warmup your Symfony cache to refresh JavaScript translations.
     *
     * If you use TypeScript, you can rename this file to "translator.ts" to take advantage of types checking.
     */

    -import { trans, getLocale, setLocale, setLocaleFallbacks } from '@symfony/ux-translator';
    +import { trans, getLocale, setLocale, setLocaleFallbacks, throwWhenNotFound } from '@symfony/ux-translator';
    import { localeFallbacks } from '../var/translations/configuration';

    setLocaleFallbacks(localeFallbacks);
    +throwWhenNotFound(true)

    export { trans }
    export * from '../var/translations';
```

@javiereguiluz do you know if we can mix diff + PHP languages highlight? 🙏🏻

Reply:
`.. code-block:: diff`

This is alread used in the symfony docs, but never in the diff+language or language+diff way.

<img width="1252" alt="Capture d’écran 2024-09-21 à 21 58 18" src="https://github.com/user-attachments/assets/1d36b529-3113-4bc4-af67-9f79bef22a23">

The .rst source: 

```rst

Once enabled, the security system redirects unauthenticated visitors to the
``login_path`` when they try to access a secured place (this behavior can
be customized using :ref:`authentication entry points <security-entry-point>`).

Edit the login controller to render the login form:

.. code-block:: diff

      // ...
    + use Symfony\Component\Security\Http\Authentication\AuthenticationUtils;

      class LoginController extends AbstractController
      {
          #[Route('/login', name: 'app_login')]
    -     public function index(): Response
    +     public function index(AuthenticationUtils $authenticationUtils): Response
          {
```